project-base/storefront/components/Blocks/StoreList/StoreListItem.tsx

```diff
@@ -48,12 +48,21 @@ export const StoreListItem: FC<StoreListItemProps> = ({ store, isSelected }) =>
         >
             <div className="flex items-center justify-between gap-3.5">
                 <div className="w-full items-center justify-between xl:flex">
-                    <div className="max-xl:mb-2.5">
+                    <div className="max-xl:mb-2.5 xl:w-[215px]">
                         <h5>{store.name}</h5>
                         <p className="mt-1.5 text-xs">
                             {store.street}, {store.postcode} {store.city}
                         </p>
                     </div>
+                    {store.distance && (
+                        <div className="max-xl:hidden">
+                            <p className="text-xs text-inputPlaceholder">
+                                {t('{{ distance }} km from you', {
+                                    distance: (store.distance / 1000).toFixed(0),
```

Comment:
< 1000m translates to 0km? Is that ok?

Reply:
it still rounds that number so if it's above 500m then this will be 1 km but I think this is edge case as hell